File: DevExpress.OutlookInspiredApp.Wpf/Controls/AdvancedNavBarGroup.cs
using System.Windows;
using DevExpress.Xpf.NavBar;

namespace DevExpress.DevAV.Controls {
    public class AdvancedNavBarGroup : NavBarGroup {
        public static readonly DependencyProperty AdvancedDisplayTextProperty =
            DependencyProperty.Register("AdvancedDisplayText", typeof(string), typeof(AdvancedNavBarGroup), new PropertyMetadata(null));
        public string AdvancedDisplayText {
            get { return (string)GetValue(AdvancedDisplayTextProperty); }
            set { SetValue(AdvancedDisplayTextProperty, value); }
        }
    }
}

File: VB/DevExpress.OutlookInspiredApp.Wpf/Controls/AdvancedNavBarGroup.vb
Imports System.Windows
Imports DevExpress.Xpf.NavBar

Namespace DevExpress.DevAV.Controls
    Public Class AdvancedNavBarGroup
        Inherits NavBarGroup

        Public Shared ReadOnly AdvancedDisplayTextProperty As DependencyProperty = DependencyProperty.Register("AdvancedDisplayText", GetType(String), GetType(AdvancedNavBarGroup), New PropertyMetadata(Nothing))
        Public Property AdvancedDisplayText() As String
            Get
                Return CStr(GetValue(AdvancedDisplayTextProperty))
            End Get
            Set(ByVal value As String)
                SetValue(AdvancedDisplayTextProperty, value)
            End Set
        End Property
    End Class
End Namespace
